dist/images/ovnkube.sh

```diff
@@ -914,9 +939,15 @@ case ${cmd} in
   "cleanup-ovn-node")
 	  cleanup-ovn-node
     ;;
+  "nb-ovsdb-raft")
+    ovsdb-raft nb ${ovn_nb_port}
+    ;;
+  "sb-ovsdb-raft")
+    ovsdb-raft sb ${ovn_sb_port}
+    ;;
   *)
 	  echo "invalid command ${cmd}"
-	  echo "valid v3 commands: ovs-server nb-ovsdb sb-ovsdb run-ovn-northd ovn-master ovn-controller ovn-node display_env display ovn_debug cleanup-ovs-server cleanup-ovn-node"
+    echo "valid v3 commands: ovs-server nb-ovsdb sb-ovsdb run-ovn-northd ovn-master ovn-controller ovn-node display_env display ovn_debug cleanup-ovs-server cleanup-ovn-node nb-ovsdb-raft sb-ovsdb-raft"
```

Comment:
@girishmg is this whitespace change intended?

Reply:
will fix it